app/Providers/AppServiceProvider.php

```diff
@@ -93,6 +93,8 @@ public function register(): void
                 return $app->make(CacheFactory::class);
             });
         });
+
+        $this->loadMigrationsFrom(database_path('migrations/Legacy'));
```

Comment:
Were Legacy migrations not running before?

Reply:
As far as I could tell, no